src/components/Identity/createManualIdSyncs.js

```diff
@@ -10,18 +10,18 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { isObject, isNumber, isNonEmptyString, find } from "../../utils";
+import { isObject, isInteger, isNonEmptyString, find } from "../../utils";
 
 const createSyncIdsByUrl = idSyncsProcessor => data => {
   const { idSyncs = [] } = data;
   const isIdSyncValid = (idSync = {}) =>
     isObject(idSync) &&
     idSync.type === "url" &&
-    isNumber(idSync.id) &&
+    isInteger(idSync.id) &&
     isObject(idSync.spec) &&
     isNonEmptyString(idSync.spec.url) &&
-    (idSync.spec.hideReferrer === 0 || idSync.spec.hideReferrer === 1) &&
-    (idSync.spec.ttlMinutes === undefined || isNumber(idSync.spec.ttlMinutes));
+    typeof idSync.spec.hideReferrer === "boolean" &&
```

Comment:
There's an `isBoolean` utility you can use.

Reply:
Thanks.